lib/networking.h

```diff
@@ -343,6 +346,10 @@ typedef struct {
 extern BOOL DP_active;
 extern BOOL TCP_active;
 extern BOOL IPX_active;
+
+// create a new non-blocking socket
+int make_nonblocking(SOCKET sock);
```

Comment:
This should probably be called something like `nw_SocketSetNonblock()` to match the rest of the functions defined here.

Reply:
It was considered but it's used on other sockets as well where the `nw_*` scheme wouldn't be appropriate.